tmk_core/common/host_driver.h

```diff
@@ -29,4 +29,5 @@ typedef struct {
     void (*send_mouse)(report_mouse_t *);
     void (*send_system)(uint16_t);
     void (*send_consumer)(uint16_t);
+    void (*send_programmable_button)(uint32_t);
```

Comment:
Would rather not use the host_driver system.

Reply:
What would be the better way?

Reply:
Not using it. Just forget it exists.

Reply:
All other similar functionality is using the host_driver, no?
Should I instead use a plain non-static function?
Would it make sense to completely remove the host_driver system?

Reply:
I will have to play around with this myself, I am fairly sure there is a way to do it without the host driver system.

Reply:
@fauxpark did you end up having a stab at this?